developer-docs/platform_introduction.md

```diff
@@ -217,70 +217,76 @@ $ minikube ip
 Visit http://<192.168.64.3>:8081
 
 ### Re-deploy with kd
-Deploying with kubectl has some limitations - in particular that it isn't possible to template out variables, making
-similar deployments to different environments very cumbersome. We use a tool called kd to overcome this constraint.
+
+Deploying with `kubectl` has some limitations - in particular that it isn't possible to template out variables, making similar deployments to different environments very cumbersome. You can use a tool called `kd` to overcome this constraint.
 
 [You can download kd here](https://github.com/UKHomeOffice/kd/releases). Please download the latest release for your computer.
 You will then need to make it executable and ideally move it to somewhere on your PATH and rename it to kd.
-```
-chmod +x kd_linux_amd64
-sudo mv kd_linux_amd64 /usr/bin/kd
+
+```bash
+$ chmod +x kd_linux_amd64
+$ sudo mv kd_linux_amd64 /usr/bin/kd
```

Comment:
should be `/usr/local/bin` (the locked down laptops that dsab got wouldn't let them write to /usr/bin